docs/how_tos/feature_toggle.rst

```diff
@@ -0,0 +1,120 @@
+Feature Toggle HOWTO
+====================
+
+
+Introduction
+------------
+
+This HOWTO explains how frontend-app-payment can use Waffle Flags for feature toggling.
```

Comment:
[nit] Specify as Django Waffle Flags?

Reply:
Nervermind, you explain that soon after!

Reply:
Do we ever explain why we use waffle flags instead of frontend feature flags via configs?

Reply:
No, we don't! Good idea, I added it. Let me know what you think. [[diff](https://github.com/openedx/frontend-app-payment/compare/6995d4bbf0f7600fbd0b0f3254659f488e464136..728ce1ec32acf343c92b0edaaf78a51bf34dfec3)]

Reply:
Thank you, looks great!  I would add too that you'd need to merge/deploy the frontend if you take that route (if it's not obvious?).

Reply:
Good add. [[diff](https://github.com/openedx/frontend-app-payment/compare/728ce1ec32acf343c92b0edaaf78a51bf34dfec3..a503a1afd812781cc5fa8769b433bde08e9c0615?short_path=c354c72#diff-c354c72baf6f4b76a4c28f36f23eaadc8fc34e5782d7349e2a1d6a2acb0c77ce)]